pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -12,16 +12,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "gitoxide";
-  version = "0.10.0";
+  version = "0.1.0";
```

Comment:
tag `v0.1.0` is from 2020-06-12

https://github.com/Byron/gitoxide/tree/v0.1.0

commit `4624725f54a34dd6b35d3632fb3516965922f60a` AKA tags: `git-index-v0.1.0`/`git-hash-v0.9.0`/`git-features-v0.19.0`/`git-bitmap-v0.0.1` is the latest tagged release

Reply:
Wait, that is weird, look at the releases page. https://github.com/Byron/gitoxide/releases/tag/git-index-v0.1.0